internal/validation/openapi.go

```diff
@@ -312,11 +313,22 @@ func Validate(ctx context.Context, outputLogger log.Logger, schema []byte, schem
 		status = "OpenAPI document valid with warnings ⚠"
 	}
 
-	reportOutput, url, err := generateReport(ctx, res)
+	reportOutput, url, digest, err := generateReport(ctx, res)
 	if err == nil && reportOutput != "" {
 		outputLogger.Info(reportOutput)
 	}
 
+	cliEvent := events.GetTelemetryEventFromContext(ctx)
+	if cliEvent != nil {
+		infoCount := int64(len(vInfo))
+		warnCount := int64(len(vWarns))
+		errCount := int64(len(vErrs))
+		cliEvent.LintReportInfoCount = &infoCount
```

Comment:
Is this the CLI execution event? If so these values will get constantly overriden as the workflow moves through all the sources it validates

Reply:
Yes it will, and that's fine. I've made it so that the TARGET_GENERATE events underneath all inherit from their parent so the lint data on these children events will make sense.

Reply:
See here https://github.com/speakeasy-api/speakeasy-core/pull/34/files